tidying

ikona23 opened the pull request:
I wrapped everything into id of the body. And continued till everything was ok :). Commenting really helped to know where am I, of course good IDE. (thank you ATOM), when I double clicked on div it will show me where is the end of the tag. Please review my work and let me know what needs to be fixed.

ikona23 committed:
tidying

guanzhou-zhao commented on pristine-zen-garden.html:
```diff
@@ -0,0 +1,299 @@
+            "Check the validity of this site&#8217;s HTML">HTML</a>
+            <a class="zen-validate-css" href=
+            "http://jigsaw.w3.org/css-validator/check/referer" title=
+            "Check the validity of this site&#8217;s CSS">CSS</a> <a class=
```
@ikona23 
About #1 , the code looks pretty good. Especially that the lines have been wrapped.

If you can put the last 3 <anchor> tags in <footer> tag to new lines, that'd be perfect and we can simulate the full cycle of "pull request".

Cheers, Ben

guanzhou-zhao commented on pristine-zen-garden.html:
```diff
@@ -0,0 +1,299 @@
+            "https://github.com/mezzoblue/csszengarden.com" title=
+            "Fork this site on Github">GH</a>
+        </footer>
+    </div>
```
@ikona23 
About #1 , the code looks pretty good. Especially that the lines have been wrapped.

If you can put the last 3 &lt;anchor&gt; tags in &lt;footer&gt; tag to new lines, that'd be perfect and we can simulate the full cycle of "pull request".

Cheers, Ben

ikona23 replied:
All done, Ben. Thanks for pointing that out.

Regards Peter

ikona23 committed:
fixed anchor tags

harrison-symes commented:
Reviewed this code, no issues as far as I can see after Ben's correction. good stuff Peter!

guanzhou-zhao committed:
Move all anchors tag to new lines in footer tag

Not necessary, just to keep all anchor tags, which in footer, are all the same starting from new lines.
